test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -400,16 +409,20 @@ def test_heatmaps_to_keypoints(self):
         assert torch.all(out2[0].eq(out_trace2[0]))
         assert torch.all(out2[1].eq(out_trace2[1]))
 
+    @skipFxTest(reason="torch._dynamo.exc.Unsupported: TorchDynamo purposely graph breaks on RNN, GRU, LSTMs")  # fmt: skip
     def test_word_language_model_RNN_TANH(self):
         self.run_word_language_model("RNN_TANH")
 
+    @skipFxTest(reason="torch._dynamo.exc.Unsupported: TorchDynamo purposely graph breaks on RNN, GRU, LSTMs")  # fmt: skip
     def test_word_language_model_RNN_RELU(self):
         self.run_word_language_model("RNN_RELU")
 
+    @skipFxTest(reason="torch._dynamo.exc.Unsupported: TorchDynamo purposely graph breaks on RNN, GRU, LSTMs")  # fmt: skip
```

Comment:
So these skips are inserted by a local script. But I couldn't figure out an easy way to make both black and flake8 happy.
I don't want to manually change the line, or tweak the script to segment the line.

This if too long is going to fail linter test.

Reply:
No choice but to multiline.. :(

Reply:
This crashes black 22.3.0 :(  
```python
...
# fmt: off
@decorator(...)
# fmt: on
...
```

Reply:
Bumped black to fix ^